src/components/team/account-ownership-transfer.tsx

```diff
@@ -53,20 +56,38 @@ const AccountOwnershipTransfer = ({accountId}: {accountId: number}) => {
             type="button"
             disabled={loading || inviteeEmail === ''}
             onClick={async () => {
+              const invitationDetails = {
+                accountId,
+                inviteeEmail,
+                ownerId: viewer?.id,
+              }
+
               try {
                 setLoading(true)
 
                 await sendOwnershipTransferInvite(inviteeEmail, accountId)
 
                 setInviteeEmail('')
 
+                track(
+                  'AccountOwnershipTransferInvite successfully sent',
```

Comment:
```suggestion
                  'sent account ownership transfer invite',
```

just a format change from "logs" to "analytics" which are generally more customer-oriented versus dev oriented